src/util/descendantThoughts.ts

```diff
@@ -2,11 +2,11 @@ import { equalPath } from './equalPath'
 import { Path } from '../types'
 
 /** Returns true if thoughts subset is contained within superset (inclusive). */
-export const subsetThoughts = (superset: Path | null, subset: Path | null) => {
+export const descendantThoughts = (superset: Path | null, subset: Path | null) => {
```

Comment:
Let's call this `isDescendantPath`. The `is` prefix will help communicate that it is a predicate.